pkg/pillar/cmd/zedrouter/lisp.go

```diff
@@ -370,8 +370,8 @@ func deleteLispConfiglet(lispRunDirname string, isMgmt bool, IID uint32,
 	EID net.IP, AppIPAddr net.IP, globalStatus types.DeviceNetworkStatus,
 	legacyDataPlane bool) {
 
-	log.Debugf("deleteLispConfiglet: %s %d %s %v\n",
-		lispRunDirname, IID, EID, globalStatus)
+	log.Debugf("deleteLispConfiglet: %s %d %s %s %v\n",
+		lispRunDirname, IID, EID, AppIPAddr, globalStatus)
```

Comment:
This feels like a change unrelated to the rest of this PR, no?

Reply:
This popped up in Yetus checks through my changes in zedbox (I touched zedbox for vaultmgr, and zedbox pulls in zedrouter). 
To make Yetus happy I had to add this change.